examples/user_guide/8_Geography.ipynb

```diff
@@ -321,6 +346,39 @@
     "      shade(dsgeo.hillshade(terrain), cmap=['black', 'white'], how='linear', alpha=128))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Proximity\n",
+    "\n",
+    "The `datashader.spatial.proximity` function operates on a given aggregate to produce a new distance aggregate based on target values and a distance metric.\n",
+    "\n",
+    "A powerful feature of `proximity` is that you can target specific values in the aggregate for distance calculation, while others are ignored.  Play with the `target_values` parameter below and see the difference of using `target_values=[1,2,3]` vs. `target_values=[2]` vs. `target_values=[3]`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from datashader.spatial import proximity\n",
+    "from datashader.colors import viridis\n",
+    "\n",
+    "df = pd.DataFrame({\n",
+    "   'x': [-10, -10, -4, -4, 1, 3, 7, 7, 7],\n",
+    "   'y': [-5, -10, -5, -5, 0, 5, 10, 10, 10]\n",
+    "})\n",
+    "\n",
+    "cvs = ds.Canvas(plot_width=W, plot_height=H, x_range=(-20, 20), y_range=(-20, 20))\n",
+    "agg = cvs.points(df, x='x', y='y')\n",
+    "proximity_agg = proximity(agg, target_values=[1, 2, 3])\n",
+    "shade(proximity_agg, cmap=viridis)"
+   ]
+  },
```

Comment:
It would be great if you could unpack this a bit more, maybe demonstrating the effects of choosing one target value and putting them side by side, like:

```
tf.Images(*[shade(proximity(agg, target_values=t, cmap=viridis, name=str(t))) for t in [[3], [3,2],[1,2,3]]) 
```

(untested code). I think this would go into a second cell, with the first cell leaving off `target_values` altogether, so that we could first explain the basic concept, and then show how to control the target values.  I also think it would be helpful to show the target values explicitly in each plot, by overlaying them in red or yellow using `tf.stack(tf.spread(shade(agg)), ...)`, as in the network plotting examples.

Reply:
Yes good idea.  I added an example of NOT supplying target values vs. using the `target_values` argument:

![image](https://user-images.githubusercontent.com/433221/63974660-e941d900-ca72-11e9-8aa4-2c4762efae63.png)

Reply:
Nice. And pretty!